model/src/main/resources/reference.conf

```diff
@@ -36,6 +36,11 @@ ort {
   }
 
   downloader {
+    // Only used with CLI with '--license-classifications-file' option.
```

Comment:
"Only used by the CLI tool when the '--license-classifications-file' option is specified." (To avoid the "with" word-doubling.)